### PR TITLE
K8S-380 Add Iceberg info to Topic CRD doc

### PR DESCRIPTION
**Pull request by JakeSCahill**
## Description

Review deadline: 2 December

Did some testing to make sure the property gets added to the topic in 24.3: https://redpandadata.atlassian.net/browse/K8S-380

## Page previews

https://deploy-preview-881--redpanda-docs-preview.netlify.app/24.3/manage/kubernetes/k-manage-topics/#configure-iceberg-topics

Note: the link to the Iceberg doc will work when #800 is merged

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/manage/pages/kubernetes/k-manage-topics.adoc [hunk 1]
```diff
@@ -126,6 +126,50 @@ The cleanup policy determines how to manage the partition log files when they re
 
 If the cleanup policy is `delete` or `compact,delete` you can <<Delete records from a topic>>.
 
+=== Configure Iceberg topics
+
```
Comment (david-yu) [inline modules/manage/pages/kubernetes/k-manage-topics.adoc:130]: Should we add a note here saying that an Enterprise license is required?

**Review by david-yu — APPROVED** [review]
LGTM assuming we add enterprise license blurb.

File: modules/get-started/pages/whats-new.adoc [hunk 2]
```diff
@@ -7,6 +7,10 @@ This topic includes new content added in version {page-component-version}. For a
 * xref:redpanda-cloud:get-started:whats-new-cloud.adoc[] 
 * xref:redpanda-cloud:get-started:cloud-overview.adoc#redpanda-cloud-vs-self-managed-feature-compatibility[Redpanda Cloud vs Self-Managed feature compatibility]
 
+== Leader pinning
```
Comment (asimms41) [inline modules/get-started/pages/whats-new.adoc:10]: There are duplicate leader pinning paragraphs in What's new.

File: modules/manage/pages/kubernetes/k-manage-topics.adoc [hunk 3]
```diff
@@ -126,6 +126,55 @@ The cleanup policy determines how to manage the partition log files when they re
...
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+
+In addition to the general prerequisites for managing topics, ensure that:
+
+- Iceberg support is enabled in your Redpanda cluster.
+- Tiered Storage is enabled in your Redpanda cluster.
```
Comment (asimms41) [inline modules/manage/pages/kubernetes/k-manage-topics.adoc]: Should Tiered Storage come first with a link to enable this feature?

**Review by asimms41 — CHANGES_REQUESTED** [review]
Looks good. Not sure how the duplicate leader pinning got in?

**Review by asimms41 — APPROVED** [review]
LGTM

**Pull request merged**